package.json

```diff
@@ -65,9 +65,9 @@
                     "@types/react": "^18.0.0"
                 }
             },
-            "@types/styled-components": {
+            "styled-components": {
                 "peerDependencies": {
-                    "@types/react": "*"
+                    "@types/react": "^18.0.0"
```

Comment:
Need this to prevent those nasty "... cannot be named without a reference to ..." errors because styled-components used `@types/react@17` somehow 🤷🏼‍♂️